app/services/doc_auth_router.rb

```diff
@@ -54,6 +54,7 @@ module DocAuthRouter
     # i18n-tasks-use t('doc_auth.errors.alerts.ref_control_number_check')
     DocAuth::Errors::REF_CONTROL_NUMBER_CHECK =>
       'doc_auth.errors.alerts.ref_control_number_check',
+    DocAuth::Errors::SELFIE_FAILURE => 'doc_auth.errors.alerts.selfie_failure',
```

Comment:
It looks like the other errors here have a comment above that references the translated content used. Should we add that here?

Or is the lack of a comment because we don't have an actual error message yet (in which case I wonder if we need this line)?

Reply:
@night-jellyfish , that will cause i18n lint errors. Still waiting on those that most likely will be available in the future, can be added back when those become available.

Reply:
Ok. I'm a little hesitant to add code we're not using yet, because I've seen that become permanently unused code in the past.

If it is marked in a ticket somewhere to come back to it, whether to add it in or delete it if we no longer need it, I'd feel more comfortable merging it.

Reply:
@night-jellyfish removed.